src/test/java/com/faforever/client/login/LoginControllerTest.java

```diff
@@ -17,15 +18,19 @@
 import static org.mockito.Mockito.when;
 
 public class LoginControllerTest extends AbstractPlainJavaFxTest {
+  private static final String FORGOT_LOGIN_URL = "http://app.faforever.com/faf/forgotPass.php";
+  private static final String CREATE_URL = "https://www.faforever.com/account/register";
```

Comment:
To avoid accidental calls to "the internet" and specific domain/URLs, use `localhost` instead